package.json

```diff
@@ -70,6 +70,9 @@
     "ts-node-dev": "^1.0.0-pre.44",
     "typescript": "^3.8.2"
   },
+  "optionalDependencies": {
```

Comment:
Couldn't install new packages without moving this to optional. Let me know if this still works on mac, if not might need a different place  to put it.

Reply:
wroked well, thanks for the note!